ui/admin/setup-edit.php

```diff
@@ -168,7 +168,7 @@
                     ),
                     'class' => 'pods-validate pods-validate-required'
                 ) ); ?>
-                    <input type="button" class="save-button button" value="OK" /> <a class="cancel" href="#cancel-edit">Cancel</a>
+                    <input type="button" class="save-button button" value="<?php _e( 'OK', 'pods' ); ?>" /> <a class="cancel" href="#cancel-edit"><?php _e( 'Cancel', 'pods' ); ?></a>
```

Comment:
Nope I have not tested this.  I don't recall offhand how to change the button text and also keep the value something not translated.  Maybe that can only happen with <button> tag and not <input type=button>